src/index.js

```diff
@@ -19,7 +19,8 @@ import { LandingPage } from './components/pages/Landing';
 import { ExampleDataViz } from './components/pages/ExampleDataViz';
 import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
-import { CaseViewPage} from './components/pages/CaseView';
+import FamilyIntake from './components/common/FamilyIntake';
+import { CaseViewPage } from './components/pages/CaseView';
 
```

Comment:
The merge conflict may be coming from line 22 on index.js. It doesn't appear to be a major issue, might just need to run git pull main from the feature branch. I think there are a few examples of this in the Git modules in Canvas, too.

Reply:
Thank you. I'm changing this right now.

Reply:
Awesome! Looks good!